dgl_sparse/src/matmul.cc

```diff
@@ -98,5 +139,34 @@ torch::Tensor SDDMMNoAutoGrad(
   return ret;
 }
 
+torch::Tensor SDDMMNoAutoGrad(
```

Comment:
It seems that by extending the function with `const std::string& op` might have changed the meaning of the function and made the name not a good fit?

(please update the PR with since #5032 has merged)